README.md

```diff
@@ -1,65 +1,126 @@
+   $ ruby --version && irb
+   ```
+If you don't have Ruby, you can find a complete list of ways to install Ruby, including using third-party tools like rvm, see:
 
-bundle install
-and
+(https://www.ruby-lang.org/en/downloads/)
+### 🛠️ Installation & Set Up
 
-rails generate rspec:install
-run the test with:
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
-rspec spec
-Author
-👤 Isigba Ambrose
 
-GitHub: @Ambrosegithub
-Twitter: @Glorious12
-🤝 Contributing
+- Open the [Repository Link](https://github.com/Ambrosegithub/Budgetary-App) of this project in GitHub.
+
+- Once the files are on your machine, open capstone-ruby folder in [Visual Studio Code](https://code.visualstudio.com/).
+
+### Install
+ Start by cloning the repository.
+ 
+- git@github.com:Ambrosegithub/Budgetary-App.git
+
+  Navigate to the created folder.
+
+- Initialize your project with Git.
+- Make sure that your project has Postgres database set up.
+- Run rails server and visi thttp://127.0.0.1:3000/ in your browser!
+
+  ### run the following in your terminal:
+  - gem install rails
+  - gem install pg
+  - bundle install (use this whenever you want add a gem in your gem file)
+  - rails db:create
+  - change your database password to suit yours in database.yml
+  
+  
+
+### Usage
+--Once you have your repository cloned on your machine, feel free to modify it as needed.
+- To add your own page
+
+### Testing with Rspec and Capybara
+  - In your terminal run > 
+     bundle exec rspec
+  - run rspec spec
+
+  
+## Authors
+👤 **Isigba Ambrose**
+
+- GitHub: [@Ambrosegithub](https://github.com/Ambrosegithub)
+- Twitter: [@Glorious12](https://twitter.com/Glorious851)
+- LinkedIn: [Ambrose Isigba](https://www.linkedin.com/in/ambrose-isigba/)
+
+
+## 🤝 Contributing
+
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the issues page.
+Feel free to check the [issues page](https://github.com/Ambrosegithub/Budgetary-App/issues).
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+## Acknowledgments
+
+Web Design
+All the template colors, font, and layout distribution for the whole project were designed by Gregoire Vella on Behance. A link to the project can be found here. Check Gregoire for more amazing GUI and Graphic designs!
+
+-Hat tip to all documentations used
+-Inspiration 
+
+
+## ❓ FAQ
+
+
+#### [Question_1] Do I need to have a program to manage the database. 
+
+##### [Answer_1] Yes you need postgesql or sqllite ,check installation online.
+
+#### [Question_2] Can I access the owner database .
 
-Show your support
-Give a ⭐️ if you like this project!
+##### [Answer_2]No because its on their local database .
 
-Acknowledgments
-Credits go to Gregoire at Behance
-📝 License
+## 📝 License
 This project is MIT licensed.
 
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] OPTIONAL
Kindly endeavor to remove the guide note below the MIT license.

Reply:
Done